content/docs/resiliency/timeouts.txt

```diff
@@ -0,0 +1,88 @@
+---
+title: Timeouts
```

Comment:
the timeout file should be a md file and the content should be different. maybe merge the latest from main?

Reply:
we had two, deleting this one